client/ayon_blender/plugins/publish/extract_local_render.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+import bpy
+import pyblish.api
+
+from ayon_core.pipeline import publish
+from ayon_blender.api import plugin
+
+
+class ExtractLocalRender(
+    plugin.BlenderExtractor,
+    publish.OptionalPyblishPluginMixin,
+):
+    """Render the sequence locally during publish when not using farm.
+
+    This extractor renders the current scene's animation to the file output
+    paths collected on the render instance. It only runs when the instance
+    is not marked for farm processing (farm=False).
+    """
+
+    label = "Extract Local Render"
+    hosts = ["blender"]
+    families = ["render"]
```

Comment:
maybe we can add a collector for this families, and this can be changed to:
```suggestion
    families = ["render.local"]
```